### PR TITLE
trtllm CI fixes

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

**Review by fpetrini15 — DISMISSED** [review]

**Review by krishung5 — CHANGES_REQUESTED** [review]
Can you please update the name of this PR for better future reference?

File: build.py [hunk 1]
```diff
@@ -1314,20 +1318,6 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
...
-            backends[be]
-        )
-
-        response = requests.get(url)
-        spec = importlib.util.spec_from_loader(
-            "trtllm_buildscript", loader=None, origin=url
-        )
-        trtllm_buildscript = importlib.util.module_from_spec(spec)
-        exec(response.content, trtllm_buildscript.__dict__)
-        df += trtllm_buildscript.create_postbuild(backends[be])
```
[the remark this answers is not in the document]
Comment (mc-nv) [reply build.py:1330]: I compared image size on disk and it looks like my latest image is much smaller than staged one.

```
$ docker image inspect ritonserver:23.12-RC.11420617-trtllm-amd64 --format {{.Size}}
14907526249

$ docker image inspect  tritonserver:23.11-trtllm-python-py3-stage --format {{.Size}}
19851316463
```

[the remark this answers is not in the document]
Comment (krishung5) [reply build.py:1330]: I noticed that, and this is great! But I think we could still drop those libs, and we would need to install some packages as mentioned here: https://gitlab-master.nvidia.com/ftp/tekit_backend/-/blob/main/tools/gen_trtllm_dockerfile.py?ref_type=heads#L94-96

[the remark this answers is not in the document]
Comment (mc-nv) [reply build.py:1330]: Explicit removal of the links won't give you the necessary result. 
To do so we need to introduce multistage container and limit to what we wan't to pull on top of the base container.
We can try, but  but in that case we need to maintain list of the files and libraries we wan't to care about. 
We also need to track any changes in this direction in future.

[the remark this answers is not in the document]
Comment (mc-nv) [reply build.py:1330]: @krishung5 - it's addressed

File: build.py [hunk 2]
```diff
@@ -608,7 +608,11 @@ def backend_cmake_args(images, components, be, install_dir, library_paths):
         cargs.append(cmake_backend_enable(be, "TRITON_ENABLE_MEMORY_TRACKER", True))
 
     cargs += cmake_backend_extra_args(be)
-    cargs.append("..")
+    if be == "tensorrtllm":
+        cargs.append("-S ../inflight_batcher_llm -B .")
+
+    else:
+        cargs.append("..")
     return cargs
```
[the remark this answers is not in the document]
Comment (mc-nv) [reply build.py:618]: I've removed it

**Comment by krishung5** [comment]
Changes lgtm! Could you re-run pre-commit to resolve the pre-commit check?

**Comment by mc-nv** [comment]
> Changes lgtm! Could you re-run pre-commit to resolve the pre-commit check? 
I'm not able to solve those issues. 
Provided report is false positive, for given lines there is no issues with indentation

**Review by krishung5 — APPROVED** [review]

**Pull request merged**